### PR TITLE
DDCNL-5706: Removed some unused defs and params

### PR DESCRIPTION
**Pull request by Christian-QA**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -12,7 +12,6 @@ addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"           % "3.12.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"       % "2.2.0")
 addSbtPlugin("com.typesafe.play" % "sbt-plugin"               % "2.8.19")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-digest"               % "1.1.4")
-addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"              % "1.5.1")
```
Comment (pascal22p) [inline project/plugins.sbt:15]: I don't think you can delete this. it compiles the scss into css

**Review by pascal22p — CHANGES_REQUESTED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

File: project/plugins.sbt (hunk 1 above)
Comment (SandeepUmredkar) [reply project/plugins.sbt]: Resolved

**Review by pascal22p — APPROVED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Review by grantrobson — APPROVED** [review]
lgtm

**Pull request merged**